charms/feast-ui/tests/integration/test_charm.py

```diff
@@ -0,0 +1,116 @@
+import logging
+from pathlib import Path
+
+import jubilant
+import yaml
+
+logger = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+IMAGE = METADATA["resources"]["oci-image"]["upstream-source"]
+CHARM_NAME = METADATA["name"]
+
+OFFLINE_STORE_APP = "offline-store"
+ONLINE_STORE_APP = "online-store"
+REGISTRY_APP = "registry"
+
+ADMISSION_WEBHOOK = "admission-webhook"
+RESOURCE_DISPATCHER = "resource-dispatcher"
+METACONTROLLER = "metacontroller-operator"
+FEAST_INTEGRATOR = "feast-integrator"
+ISTIO_GATEWAY = "istio-gateway"
+ISTIO_PILOT = "istio-pilot"
```

Comment:
suggestion: use the new style implemented by @orfeas-k in https://github.com/canonical/bundle-kubeflow/issues/1256 to standardize our test dependencies, for example see https://github.com/canonical/dex-auth-operator/pull/259